urbanopt_to_alfalfa.ipynb

```diff
@@ -0,0 +1,300 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import shutil\n",
+    "import json\n",
+    "import pandas\n",
+    "import datetime\n",
+    "import time\n",
+    "from alfalfa_client.alfalfa_client import AlfalfaClient\n",
+    "from pprint import pprint\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f892da4d",
+   "metadata": {},
+   "source": [
+    "### Define alfalfa client object\n",
+    "\n",
+    "Use `pip install alfalfa-client` prior to running further commands"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97e6c517",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ac = AlfalfaClient(host='https://cctwin.nrel.gov')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "79c10276",
+   "metadata": {},
+   "source": [
+    "### Define Alfalfa simulation parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "414e3ec4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you are using historian, you will need to search for this time period in Grafana dashboard to view results.\n",
+    "start_dt = datetime.datetime(2021, 7, 1, 12, 2, 0)\n",
+    "end_dt = datetime.datetime(2021, 7, 3, 0, 0, 0)\n",
+    "\n",
+    "# For external_clock == true, API calls are used to advance the model.  \n",
+    "# If external_clock == false, Alfalfa will handle advancing the model according to a specified timescale (timescale 1 => realtime)\n",
+    "params = {\n",
+    "    \"external_clock\": True,\n",
+    "    \"start_datetime\": start_dt,\n",
+    "    \"end_datetime\": end_dt\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f10a562c",
+   "metadata": {},
+   "source": [
+    "### Create folders to upload to Alfalfa from the URBANopt Project\n",
+    "\n",
+    "The code below will create folders for each URBANopt building model that can be uploaded to Alfalfa. The folder contains: \n",
+    "\n",
+    "    - Model Folder: Contains OpenStudio model (.osm file) for each building created using the URBANopt simulation.\n",
+    "    - Measures Folder: Measures to be added while running Alfalfa in the OpenStudio Workflow\n",
+    "    - Weather Folder: Contains EPW weather file \n",
+    "    - workflow.osw file: OpenStudio workflow file\n",
+    "\n",
+    "**Define the following variables before running the code**:\n",
+    "\n",
+    "- `uo_folder` : URBANopt Scenario Directory \n",
+    "- `weather`: Define weather file used in URBANopt project\n",
+    "- `workflow`: OpenStudio workflow file name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83836f8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set URBANopt scenario directory\n",
+    "uo_folder = Path('C:/a/baseline_stochastic/run/baseline_scenario_stochastic')\n",
```

Comment:
can we include an example scenario in this repository? So people can just download the repo and use it right away?